scalableminds/webknossos
# Mag information is not lost on empty volumes

### URL of deployed dev instance (used for testing):
- https://emptyvolumesdownloadupload.webknossos.xyz

### Steps to test:
- Select a dataset with multiple mags
- Create a volume with fewer mags
- Download that volume
- Reload the dataset
- Upload the volume
- Ensure that only the selected mags are available in the volume

### Issues:
- fixes #6128

The original handling is still largely there to handle previously exported files.

------
(Please delete unneeded items, merge only when none are left open)
